data/templates/Detector_ContainerRegistry.csx

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Threading;
+
+private static string GetQuery(OperationContext<ArmResource> cxt)
+{
+    return
+    $@"
+		let startTime = datetime({cxt.StartTime});
+		let endTime = datetime({cxt.EndTime});
+		cluster('ClusterName').database('DBName').YOUR_TABLE_NAME
+		| where Timestamp == startTime and Timestamp == endTime
```

Comment:
change this line to following ?
| where TIMESTAMP >= startTime and TIMESTAMP <= endTime

Reply:
done